test/OrchardCore.Tests/Orchard.Queries/SqlParserTests.cs

```diff
@@ -31,7 +32,7 @@ private string FormatSql(string sql)
         [InlineData("select distinct a", "SELECT DISTINCT [a];")]
         public void ShouldParseSelectClause(string sql, string expectedSql)
         {
-            var result = SqlParser.TryParse(sql, _defaultDialect, _defaultTablePrefix, null, out var rawQuery, out var messages);
+            var result = SqlParser.TryParse(sql, _configuration, _defaultDialect, _defaultTablePrefix, null, out var rawQuery, out var messages);
```

Comment:
Pass the schema directly, this is what is done for dialect and prefix.

Reply:
And then the test can verify that the schema is used in the result, like we check the prefix.

Reply:
> like we check the prefix

Where?!! we are checking the result and the expected SQL